topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.md

```diff
@@ -694,16 +698,16 @@ Congratulations! You should have a file with all the unique gene names and a cou
 
 One of the most important features of Galaxy comes at the end of an analysis. When you have published striking findings, it is important that other researchers are able to reproduce your in-silico experiment. Galaxy enables users to easily share their workflows and histories with others.
 
-To share a history, click on the {% icon galaxy-gear %} (gear) symbol in the history pane and select `Share or Publish`. On this page you can do 3 things:
+To share a history, click on the {%  icon galaxy-history-options %} history options and select `Share or Publish`. On this page you can do 3 things:
 
 
 1. **Make accessible via Link**
 
     This generates a link that you can give out to others. Anybody with this link will be able to view your history.
 
-2. **Publish History**
+2. **Make History publicly available in Published Histories**
 
-    This will not only create a link, but will also publish your history. This means your history will be listed under `Shared Data → Published Histories` in the top menu.
+    This will not only create a link, but will also publish your history. This means your history will be listed under ` Data → Histories → Published Histories` in the top menu.
```

Comment:
```suggestion
    This will not only create a link, but will also publish your history. This means your history will be listed under `Data → Histories → Published Histories` in the top menu.
```